packages/context/src/inject.ts

```diff
@@ -331,21 +395,28 @@ export function describeInjectedArguments(
   return meta || [];
 }
 
-function resolveByTag(
+function resolveBindings(
   ctx: Context,
-  injection: Readonly<Injection>,
+  bindings: Readonly<Binding>[],
   session?: ResolutionSession,
 ) {
-  const tag: string | RegExp = injection.metadata!.tag;
-  const bindings = ctx.findByTag(tag);
-
   return resolveList(bindings, b => {
     // We need to clone the session so that resolution of multiple bindings
     // can be tracked in parallel
     return b.getValue(ctx, ResolutionSession.fork(session));
   });
 }
 
+function resolveByTag(
```

Comment:
Wasn't this resolver already added by a previous pull request? I remember seeing this change in other places. Or is it just the way how git is computing the diff?